### DISCUSSION TITLE
结合url-test和load balance

### DISCUSSION THREAD

**Discussion by ky0utarou** [MetaCubeX/mihomo#1345] [Q&A]
求问是否有任何方法能结合两者？
例如动态地在延迟最低的若干节点中进行load balance

**Comment by apoiston** [comment]
这是我目前现在用的，test主Primary节点和第二Second节点后balance，给你参考一下，欢迎分享其它方法
![image](https://github.com/MetaCubeX/mihomo/assets/24912536/22eb5b6e-5509-4872-974a-56eb20d45d42)

**Reply by ky0utarou** [reply]
你这个写法看的我愣了一下
em这种手动分组倒是个近似

**Reply by apoiston** [reply]
> 你这个写法看得我愣了一下， 这种手动分组倒是一个方法

何为愣了一下？我这个写法是结合了url-test和load-balance的，我有哪里没有表达清楚的地方，你可以提出来互相交流哦

**Reply by ky0utarou** [reply]
我是指anchor和其他的syntax，之前没见过这种写法，搜了一下才知道
